GliaWidgets/SecureConversations/SecureConversations.Coordinator.Environment.swift

```diff
@@ -52,8 +52,8 @@ extension SecureConversations.Coordinator {
         var alertManager: AlertManager
         var queuesMonitor: QueuesMonitor
         var createEntryWidget: EntryWidgetBuilder
-        var shouldShowLeaveSecureConversationDialog: Bool
-        var leaveCurrentSecureConversation: Cmd
+        var shouldShowLeaveSecureConversationDialog: () -> Bool
+        var leaveCurrentSecureConversation: Command<Bool>
```

Comment:
Maybe it makes sense  to document what this `Bool` value indicates?

Reply:
Added d334d5935c4365303ed8c60433c75c3a9e952340